### ISSUE TITLE
Avoid subtitle dialog when there's just one option

### ISSUE THREAD

**Issue by alucasr** [FaisalUmair/udemy-downloader-gui#142]
It would be great to avoid user's action when there's just one subtitle.
It could be an option (maybe save subtitles, or save default subtitles... or even better: allow to indicate several subtitles (I want english, automatic english and spanish, automatic spanish))

**Comment by FaisalUmair** [comment]
Yeah, you are right. We can avoid the user action if only one subtitle is detected. But sometimes some video lectures have proper subtitles and some have auto-generated in that case, the user needs to select the subtitle which is common in most lectures of the course.

**Comment by alucasr** [comment]
The application, in order to be perfect for me :-D, it should include an option to configure which subtitles you want (in my case, that would be english, automatic english, spanish and automatic spanish), and should allow you to download all your courses without intervenction, one by one (it would not start a course download until it finishes the previous one)
Now I do this manually and it requires my attention for a very long time (I have a lot of courses)
It's already a great app, an AMAZING app... and I this could be a good improvement in order to facilitate the use (the work would require less attention)
Please, thing about it
